.github/workflows/semver-auto.yaml

```diff
@@ -0,0 +1,37 @@
+name: Add PR semver labels on push
+on: [pull_request_target]
```

Comment:
Will this workflow not be triggered by any comment, label and such? I thought `types: [synchronize]` was needed here

Reply:
me too, let me re-add it. I'm not sure why I removed it.